### PR TITLE
docs(key-concepts-update) Updated Beta and added Stable

### PR DESCRIPTION
**Pull request by GayleNeumann**
-- Updated Beta definitions and added Stable definition.
-- These definitions have been reviewed by Reza, Jimmy/Customer Support and Steven C.

### CODE REVIEW

**Comment by GayleNeumann** [comment]
Should "beta" be capitalized or lower-case in the definition? To be consistent with the rest of Kong docs.

File: app/enterprise/1.5.x/introduction/key-concepts.md [hunk 1]
```diff
@@ -20,7 +20,14 @@ API gateway authentication is an important way to control the data that is allow
 Authorization is the system of defining access to certain resources. In Kong Enterprise, Role-Based Access Control (RBAC) is the main authorization mode. To define authorization to an API, it is possible to use the ACL Plugin in conjunction with an authentication plugin.
 
 ## Beta
-A Beta designation in Kong software means the functionality of a feature is of high-quality and can be deployed in a non-production environment. **A Beta feature should not be deployed in a production environment.** Some level of support is available for the feature, although user documentation might not be of the highest quality. A Beta feature is made available to the general public for usability testing and to gain feedback about the feature before releasing it as production-ready.
+A Beta designation in Kong software means the functionality of a feature or release version is of high-quality and can be deployed in a non-production environment. Note the following when using a Beta feature or version:
```
Comment (tlblessing) [inline app/enterprise/1.5.x/introduction/key-concepts.md:23]:
```suggestion
A Beta designation in Kong software means the functionality of a feature or release version is of high quality and can be deployed in a non-production environment. Note the following when using a Beta feature or version:
```

File: app/enterprise/1.5.x/introduction/key-concepts.md [hunk 2]
```diff
@@ -20,7 +20,14 @@ API gateway authentication is an important way to control the data that is allow
...
+
+* **A Beta feature or version should not be deployed in a production environment.** 
+* Beta customers are encouraged to engage Kong Support to report issues encountered in Beta testing.  Support requests should be filed with normal priority, but contractual SLA’s will not be applicable for Beta features.
+* Support is not available for data recovery, rollback, or other tasks when using a Beta feature or version. 
+* User documentation might not be of the highest quality.
```
Comment (tlblessing) [inline app/enterprise/1.5.x/introduction/key-concepts.md:28]:
```suggestion
* User documentation might not be complete or reflect complete functionality. 
```

File: app/enterprise/1.5.x/introduction/key-concepts.md [hunk 3]
```diff
@@ -86,6 +93,9 @@ A Service, also referred to as a Service object, is the upstream APIs and micros
 
 Before you can start making requests against a Service, you need to add a [Route](#route) to it. Routes specify how (and if) requests are sent to their Services after they reach Kong. A single Service can have many Routes. After configuring the Service and the Route, you’ll be able to make requests through Kong using them.
 
+## Stable 
+A Stable release designation in Kong software means the functionality of the version is high-quality, production-ready, and released as general availability (GA). The version has been thoroughly tested, considered reliable to deploy in a production environment, and is fully supported. If updates or bug fixes are required, a patch version or minor release version is issued and fully supported.
```
Comment (tlblessing) [inline app/enterprise/1.5.x/introduction/key-concepts.md:97]:
```suggestion
A Stable release designation in Kong software means the functionality of the version is of high quality, production-ready, and released as general availability (GA). The version has been thoroughly tested, considered reliable to deploy in a production environment, and is fully supported. If updates or bug fixes are required, a patch version or minor release version is issued and fully supported.
```

File: app/enterprise/1.5.x/introduction/key-concepts.md (hunk 2 above)
Comment (tlblessing) [reply app/enterprise/1.5.x/introduction/key-concepts.md]: OR User documentation might not be complete or reflect all available functionality.

**Pull request merged**